templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,18 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 58**
+
+* [`GET /users`](/api/get-users), [`GET /users/{user_id}`](/api/get-user),
+  [`GET /users/{email}`](/api/get-user-by-email)and [`GET /users/me`](/api/get-own-user):
+  User objects now contain the `is_moderator` field as well.
+
+* [`GET /events`](/api/get-events): Added `is_moderator` field to
+  person objects sent in `realm_user` events.
+
+* [`POST /register`](/api/register-queue): Added `is_moderator` field
+  in the user objects of `realm_users` dict.
```

Comment:
Do you think it's a good idea to keep adding fields like this to the API?  In theory, the information is already encoding in `role`, and client developers can just compute it using the `role` field.  (I realize `is_admin` is a convenience, as is `is_guest`, but overall these fields make life more complicated and require these decisions about whether we're going to set `is_moderator` to true or false for an administrator, etc.)

@gnprice thoughts?

Reply:
Oh, do we not currently send `role` for other users?  Maybe my proposal is that we should make this commit instead add `role` to the API details around other users, and we can think about a broader API migration to start just sending `role` rather than these N similar fields down the line.  (Probably will need a `client_capability` for that).

And then we can just patch `_add_user` in `people.js` to translate the `role` field into the `is_moderator` property so that you can write the frontend code as written in this PR.